Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,87 @@
 ﻿using System;
+using System.Globalization;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            Console.Write("Which decimal do you want to convert to binary: ");
```

Comment:
Have 3 methods instead of 3 blocks of code.

Reply:
solved